ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+interpreter_python      = auto_silent
+log_path                = /var/log/cray/ansible.log
```

Comment:
Can we take ANSIBLE_FORCE_COLOR=1 PYTHONUNBUFFERED=1 ANSIBLE_STDOUT_CALLBACK=debug
out of the packer config and add the equivalent here too?
Definitely no reason to hold up merging for it though.

Reply:
Think so. I do want to rebuild after merging this to main.

Reply:
I also wanted to enable color for packer, which is disabled in the CSM shared lib. Maybe we can make a follow up Jira for those two items (ansible.cfg and csm-shared-library).

Reply:
I'll just make the changes with this Jira. Lazy.

Reply:
I'll add ANSIBLE_FORCE_COLOR but I'll leave the other two, I can't move PYTHONUNBUFFERED but I also don't want to change the runtime Ansible too much.